specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/keys.json

```
@@ -1311,6 +1310,20 @@
       },
       "description": "The key item containing key metadata."
     },
+    "KeyReleasePolicy": {
+      "properties": {
+        "contentType": {
+          "description": "Content type and version of key release policy",
+          "type": "string",
+          "default": "application/json; version=1.0"
```

Since you already treat it as UTF8 anyway, it would help inform people if you made this, perhaps:
```suggestion
          "default": "application/json; charset=utf-8; version=1.0"
```